text/0006-shipyard-next-generation.md

```diff
@@ -0,0 +1,317 @@
+# The next generation of Shipyard
+
+The next generation of Shipyard to allow custom delivery/remediation processes.
+
+## Motivation
+
+*There's always room for smart enhancements* - the Shipyard specification is no exception.
+
+A shipyard allows a **Site Reliability Engineer (SRE)** to specify WHAT should happen in the course of an artifact delivery or remediation process. Due to certain use cases, it has been identified that the current specification of the shipyard has limitations and hides details from the SRE. This KEP proposes an enhancement of the *Shipyard* specification to bring flexibility into the definition of the *WHAT*, but still provides an opinionated approach and smart defaults.
```

Comment:
> A shipyard allows a **Site Reliability Engineer (SRE)**
As this is literally the intro to this topic, it sounds like Shipyard is something that will be primarily used by SREs.

I would remove the SRE from this introduction completely, as our end-users are not exclusively SREs, but also Developers, DevOps, etc....

> Due to certain use cases, it has been identified ...
This sounds like a weak motiviation, it basically accuses "certain use cases"...

Here is my take on this first paragraph:

The current specification of [Keptns Shipyard](https://github.com/keptn/spec/blob/0.1.7/shipyard.md) has several limitations and hides details from end-users. This KEP provides an enhancement of the *Shipyard* format such that it allows end-users to specify *WHAT* should happen in the course of an artifact delivery or remediation process. While it brings flexibility into the definition of the *WHAT*, it is still an opinionated approach with smart defaults.

Reply:
Thanks for improving the motivation. I added a slight modification but I like your version better. I use it now. :)